src/argument_parsing/build.cpp

```diff
@@ -151,6 +170,8 @@ void run_build(seqan3::argument_parser & parser)
     size_t size{};
     std::from_chars(arguments.size.data(), arguments.size.data() + arguments.size.size() - 1, size);
     size *= multiplier;
+
+    // !! core dumped
     arguments.bits = size / (((arguments.bins + 63) >> 6) << 6);
```

Comment:
Program exited with arithmetic exception (core dumped). I think the trouble comes when the IBF doesn't have power of two number of bins. Please take a look @marehr  

gdb output:
`Program received signal SIGFPE, Arithmetic exception.
0x00005555555bf39b in sliding_window::app::run_build (parser=...) at /home/evelin/metagenomics/sliding-window/src/argument_parsing/build.cpp:174
174         arguments.bits = size / (((arguments.bins + 63) >> 6) << 6);`

Reply:
Actually the issue was not from bin numbers. Still looking for cause

`Program received signal SIGSEGV, Segmentation fault.
0x00005555555c81e4 in std::vector<std::filesystem::__cxx11::path, std::allocator<std::filesystem::__cxx11::path> >::operator[] (this=0x0, __n=0)
    at /usr/include/c++/10/bits/stl_vector.h:1046
1046            return *(this->_M_impl._M_start + __n);
`